docs/device-agent/deploy.md

```diff
@@ -28,8 +28,9 @@ to the registered devices when you're happy with what you've created.
 The device agent does not allow local access to the Node-RED editor. This ensures
 the device is running the deployed snapshot without modification.
 
-When running on FlowForge Cloud, or a premium licensed FlowForge instance, a
-device can be placed in Developer Mode that enables remote access to the editor.
+When running on FlowForge Cloud, or a premium licensed FlowForge instance (with the
+MQTT broker enabled), a device can be placed in Developer Mode that enables remote 
```

Comment:
link here to documentation setting that up?

Reply:
It's slightly different for each driver. Docker/Kube set it up by default, but localfs is a bit complex https://flowforge.com/docs/install/local/#setting-up-mosquitto-(optional)